src/app/landing-page/landing-page.component.html

```diff
@@ -0,0 +1,6 @@
+
+<p>{{emailInput}}</p>
+<input type="text" [(ngModel)]="emailInput">
+<button (click)="submitEmail(emailInput)">
+  submit email
```

Comment:
In a separate PR please integrate https://github.com/ngx-translate/core to store labels outside of the code base.